stellargraph/data/explorer.py

```diff
@@ -854,13 +853,17 @@ def run(
         failures = 0
 
         def not_progressing_enough():
-            # (this uses Beta(1, 1) as the prior, since it's uniform on p)
+            # Estimate the probability p of a walk being long enough; the 95% percentile is used to
+            # be more stable with respect to randomness. This uses Beta(1, 1) as the prior, since
+            # it's uniform on p
             posterior = stats.beta(1 + successes, 1 + failures)
```

Comment:
Minor thing, but since we're only querying the `ppf`, it turns out to be much faster to do that directly rather than construct the distribution object:
```python
from scipy import stats
%timeit -n 100 stats.beta.ppf(0.95, 11, 10)
# 198 µs ± 18.9 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)
%timeit -n 100 stats.beta(11, 10).ppf(0.95)
# 674 µs ± 31.9 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)
```

As such:

```suggestion
            posterior = stats.beta.ppf(0.95, 1 + successes, 1 + failures)
```

and `return posterior < p_walk_success_threshold`.